packages/extensions-sdk/src/cli/commands/validators/check-built-code.ts

```diff
@@ -0,0 +1,53 @@
+import { EXTENSION_PKG_KEY } from '@directus/extensions';
+import fse from 'fs-extra';
+import { type Ora } from 'ora';
+import type { Report } from '../../types.js';
+
+const checkBuiltCode = {
+	name: 'built-code',
+	handler: async (spinner: Ora, reports: Array<Report>) => {
+		spinner.text = 'Check for built code';
+
+		let codePath = '/dist';
+
+		if (await fse.pathExists(`${process.cwd()}/package.json`)) {
```

Comment:
This might fail on a Windows, since this is using forward slashes? 

We should use `path.resolve('package.json')` and `path.resolve(codePath)` further down to obtain the actual paths, like we do in the `add` command for example: 

https://github.com/directus/directus/blob/3e1265db8b19ba63596c442342122f804eaeb3b9/packages/extensions-sdk/src/cli/commands/add.ts#L35